Source/inv_iterators.hpp

```diff
@@ -261,6 +261,37 @@ class BeltPlayerItemsRange {
 	Player *player_;
 };
 
+/**
+ * @brief A range over non-equipped player items in the following order: Inventory, Belt.
+ */
+class InventoryAndBeltPlayerItemsRange {
+public:
+	explicit InventoryAndBeltPlayerItemsRange(Player &player)
+	    : player_(&player)
+	{
+	}
+
+	[[nodiscard]] ItemsContainerListRange::Iterator begin() const // NOLINT(readability-identifier-naming)
+	{
+		return ItemsContainerListRange::Iterator({
+		    InventoryPlayerItemsRange(*player_).begin(),
+		    BeltPlayerItemsRange(*player_).begin(),
+		});
+	}
+
+	[[nodiscard]] ItemsContainerListRange::Iterator end() const // NOLINT(readability-identifier-naming)
+	{
+		auto result = ItemsContainerListRange::Iterator({
+		    InventoryPlayerItemsRange(*player_).end(),
+		    BeltPlayerItemsRange(*player_).end(),
+		});
+		return result;
+	}
+
+private:
+	Player *player_;

```

Comment:
Why does the field name end in an `_`? Shouldn't the underscore be _before_ the name to represent private data instead of _after_ it?

Reply:
After: https://github.com/diasurgical/devilutionX/blob/7771a08d033335e8857687dd0da258280908d345/Source/.clang-tidy#L70